rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -149,6 +150,20 @@
   }
 
   /**
+   * Create an observable of {@link ViewTreeObserverDrawEvent} for draws on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnDrawListener} to observe
+   * draws. Multiple observables can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<ViewTreeObserverDrawEvent> drawEvents(@NonNull View view) {
```

Comment:
Is the event object useful? I think I need to clarify in the `README.md` about when it should be used...

I think events are only useful when the listener has more than one callback parameter that you _might_ be interested in. For example, an adapter view item click gives you the position and ID of the item. In this case, though, you really only care that the event happened. As such, we can either use `Void` or emit the `View` instance.

I think we should probably use `Void` here since the callback doesn't even pass the `View` back to you, and you can always pull the view from the outer scope.